cortix/src/module.py

```diff
@@ -13,18 +13,21 @@ class Module:
     '''
 
     def __init__(self):
-        self.rank = None
+
+        self.use_mpi = False
         self.ports = []
 
     def send(self, data, port):
         '''
         Send data through a given port.
         '''
+
         if isinstance(port, str):
-            matches = [p for p in self.ports if p.name == port]
```

Comment:
This test has been eliminated because the get_port() method prevents this to happen. Also the send()/recv() methods are called many, many times and need to be lean.